src/Cake.Common/Tools/GitReleaseManager/Label/GitReleaseManagerLabeller.cs

```diff
@@ -128,31 +127,20 @@ private ProcessArgumentBuilder GetArguments(string token, string owner, string r
             builder.Append("--token");
             builder.AppendQuoted(token);
```

Comment:
Secret

Reply:
Yes, that is a great catch!  Not sure how/why this wasn't like this already?!?